containers/datalab/config/ipython.py

```diff
@@ -28,3 +28,7 @@
 
 # Enable matplotlib renderings to show up inline in the notebook.
 c.InteractiveShellApp.matplotlib = 'inline'
+
+# Customize the Save as HTML handling
+c.TemplateExporter.template_path = ['/datalab/nbconvert']
+c.TemplateExporter.template_file = 'as_html'
```

Comment:
Somewhat of a nit -- but as_html as a file name seems odd. Just 'html'? But maybe there is more to it, in which case its fine.